sceptre/scipool/config/develop/sc-tag-options.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
 sceptre_user_data:
   Departments: !file sc-tag-options/internal/Departments.json
   Projects: !file sc-tag-options/internal/Projects.json
-  CostCenters: !file https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.6/tags/SageFinancialProgramCodes.json
+  CostCenters: !rcmd 'wget -qO- https://mips-api.finops.sageit.org/tags?limit=24'
```

Comment:
i don't think this is going to work because `!rcmd` will pass the response into `CostCenter` as a string. We are expecting CostCenter to be a list in templates/sc-tag-options.j2

Reply:
Oh, I didn't realize the file resolver was doing implicit deserialization. hrm.